components/entitlement/org.wso2.carbon.identity.entitlement/src/test/resources/dbscripts/h2.sql

```diff
@@ -0,0 +1,1570 @@
+CREATE TABLE IF NOT EXISTS IDN_BASE_TABLE (
+            PRODUCT_NAME VARCHAR (20),
+            PRIMARY KEY (PRODUCT_NAME)
+);
+
+INSERT INTO IDN_BASE_TABLE values ('WSO2 Identity Server');
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH_CONSUMER_APPS (
+            ID INTEGER NOT NULL AUTO_INCREMENT,
+            CONSUMER_KEY VARCHAR (255),
+            CONSUMER_SECRET VARCHAR (2048),
+            USERNAME VARCHAR (255),
+            TENANT_ID INTEGER DEFAULT 0,
+            USER_DOMAIN VARCHAR(50),
+            APP_NAME VARCHAR (255),
+            OAUTH_VERSION VARCHAR (128),
+            CALLBACK_URL VARCHAR (2048),
+            GRANT_TYPES VARCHAR (1024),
+            PKCE_MANDATORY CHAR(1) DEFAULT '0',
+            PKCE_SUPPORT_PLAIN CHAR(1) DEFAULT '0',
+            APP_STATE VARCHAR (25) DEFAULT 'ACTIVE',
+            USER_ACCESS_TOKEN_EXPIRE_TIME BIGINT DEFAULT 3600,
+            APP_ACCESS_TOKEN_EXPIRE_TIME BIGINT DEFAULT 3600,
+            REFRESH_TOKEN_EXPIRE_TIME BIGINT DEFAULT 84600,
+            ID_TOKEN_EXPIRE_TIME BIGINT DEFAULT 3600,
+            CONSTRAINT CONSUMER_KEY_CONSTRAINT UNIQUE (TENANT_ID, CONSUMER_KEY),
+            PRIMARY KEY (ID)
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH2_SCOPE_VALIDATORS (
+	APP_ID INTEGER NOT NULL,
+	SCOPE_VALIDATOR VARCHAR (128) NOT NULL,
+	PRIMARY KEY (APP_ID,SCOPE_VALIDATOR),
+	FOREIGN KEY (APP_ID) REFERENCES IDN_OAUTH_CONSUMER_APPS(ID) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS IDN_OAUTH1A_REQUEST_TOKEN (
+            REQUEST_TOKEN VARCHAR (512),
+            REQUEST_TOKEN_SECRET VARCHAR (512),
+            CONSUMER_KEY_ID INTEGER,
+            CALLBACK_URL VARCHAR (2048),
+            SCOPE VARCHAR(2048),
+            AUTHORIZED VARCHAR (128),
+            OAUTH_VERIFIER VARCHAR (512),
+            AUTHZ_USER VARCHAR (512),
+            TENANT_ID INTEGER DEFAULT -1,
```

Comment:
Shall we maintain the default tenant ID consistent  0 or -1. There is another default of 0

Reply:
This is the same script we have in the latest IS snaspshot build and here I used the same script to initialized in memory h2 database for unit test cases.